app/public/styles/rct/custom/components/run-quality.css

```diff
@@ -21,3 +21,13 @@
     background-color: var(--red) !important;
     color: var(--white);
 }
+
+.rct .run-quality.mixed {
+    background-color: var(--orange) !important;
+    color: var(--white);
+}
+
+.rct .run-quality.NoData {
```

Comment:
It is not good to mix kebab-case and PascalCase

Reply:
apllied